html4j-maven-plugin/src/main/java/org/netbeans/html/mojo/ProcessJsAnnotationsMojo.java

```diff
@@ -54,146 +39,39 @@
 public final class ProcessJsAnnotationsMojo extends AbstractMojo {
     @Component
     private MavenProject prj;
-    
+
     @Parameter(defaultValue = "${project.build.directory}/classes")
     private File classes;
-    
+
     public ProcessJsAnnotationsMojo() {
     }
 
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
-        LinkedList<URL> arr = new LinkedList<URL>();
+        ProcessJsAnnotations jsa = new ProcessJsAnnotations() {
+            @Override
+            protected void log(String msg) {
+                getLog().info(msg);
+            }
+        };
         boolean foundAsm = false;
         for (Artifact a : prj.getArtifacts()) {
             final File f = a.getFile();
             if (f != null) {
                 if (a.getArtifactId().equals("asm")) {
```

Comment:
This is the Maven way of detecting `asm`. It makes slightly more sense than the Gradle version.